rho/scancommand.py

```diff
@@ -283,6 +283,25 @@ def _validate_options(self):
                 self.parser.print_help()
                 sys.exit(1)
 
+        # perform fact validation
+        facts = self.options.facts
+        default_facts = False
+        if facts == ['default']:
+            default_facts = True
+        elif os.path.isfile(facts[0]):
+            facts_to_collect = _read_in_file(facts[0])
+        else:
+            assert isinstance(facts, list)
+            facts_to_collect = facts
+        # check facts_to_collect is subset of utilities.DEFAULT_FACTS_TUPLE
+        all_facts = utilities.DEFAULT_FACTS_TUPLE
+        if not default_facts and not set(facts_to_collect).issubset(all_facts):
```

Comment:
`spit_results.py` will always receive and expanded list. It does still work that way.

The validation code (in `_validate_options()`)exists and is executed prior to the fact expansion code (in `_do_command()`)